README.md

```diff
@@ -17,107 +14,139 @@ A [Django](https://www.djangoproject.com/) integration for [Graphene](http://gra
 
-    def resolve_users(self, info):
-        return UserModel.objects.all()
+## License
 
-schema = graphene.Schema(query=Query)
-```
+Graphene-Django is released under the [MIT License](https://github.com/graphql-python/graphene-django/blob/master/LICENSE).
 
-Then you can query the schema:
+## Resources
 
-```python
-query = '''
-    query {
-      users {
-        name,
-        lastName
-      }
-    }
-'''
-result = schema.execute(query)
-```
+* [Official GitHub Repository](https://github.com/graphql-python/graphene-django)
+* [Graphene Documentation](http://docs.graphene-python.org/en/latest/)
+* [Django Documentation](https://docs.djangoproject.com/en/stable/)
+* [GraphQL Specification](https://spec.graphql.org/)
+* [GraphiQL](https://github.com/graphql/graphiql) - An in-browser IDE for exploring GraphQL APIs
+* [Graphene-Django Community](https://spectrum.chat/graphene) - Join the community to discuss questions and share ideas related to Graphene-Django
 
-To learn more check out the following [examples](examples/):
+## Tutorials and Examples
 
-* **Schema with Filtering**: [Cookbook example](examples/cookbook)
-* **Relay Schema**: [Starwars Relay example](examples/starwars)
+* [Official Graphene-Django Tutorial](https://docs.graphene-python.org/projects/django/en/latest/tutorial-plain/)
+* [Building a GraphQL API with Django and Graphene-Django](https://www.howtographql.com/graphql-python/0-introduction/)
+* [Real-world example: Django, Graphene, and Relay](https://github.com/graphql-python/swapi-graphene)
 
+## Related Projects
 
-## GraphQL testing clients
- - [Firecamp](https://firecamp.io/graphql)
- - [GraphiQL](https://github.com/graphql/graphiql)
+* [Graphene](https://github.com/graphql-python/graphene) - A library for building GraphQL APIs in Python
+* [Graphene-SQLAlchemy](https://github.com/graphql-python/graphene-sqlalchemy) - Integration between Graphene and SQLAlchemy, an Object Relational Mapper (ORM) for Python
+* [Graphene-File-Upload](https://github.com/lmcgartland/graphene-file-upload) - A package providing an Upload scalar for handling file uploads in Graphene
+* [Graphene-Subscriptions](https://github.com/graphql-python/graphene-subscriptions) - A package for adding real-time subscriptions to Graphene-based GraphQL APIs
 
+## Support
 
-## Contributing
+If you encounter any issues or have questions regarding Graphene-Django, feel free to [submit an issue](https://github.com/graphql-python/graphene-django/issues/new) on the official GitHub repository. You can also ask for help and share your experiences with the Graphene-Django community on [💬 Discord](https://discord.gg/Fftt273T79)
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+To keep up to date with the latest news, updates, and releases, follow Graphene-Django on [Twitter](https://twitter.com/graphene_python) and [GitHub](https://github.com/graphql-python/graphene-django).
```

Comment:
Looks like https://twitter.com/graphene_python leads to "This account doesn’t exist" for me. Remove this reference?